### ISSUE TITLE
Delete all cartridges

### ISSUE THREAD

**Issue by shinethazhava** [SqrTT/prophet#99]
After enabling Prophet: Enable Upload all files are deleted from the sandbox.
Output showing like

Deleting local zip
[test_cat] Zipping
[test_cat] Deleting remote zip
[test_cat] Sending zip to remote
[test_cat] Deleting local zip...
[test_cat] Unzipping remote zip
[test_cat] Deleting remote zip...
Cartridges uploaded successfully

**Comment by SqrTT** [comment]
So what exactly issue is?

**Comment by shinethazhava** [comment]
The issue is after enabling the out put showing properly. But my sandbox become empty

**Comment by shinethazhava** [comment]
![cap](https://user-images.githubusercontent.com/6826291/38313996-f2824f18-3842-11e8-8bbc-38a72d5f7d51.png)
![cap1](https://user-images.githubusercontent.com/6826291/38314103-225d868a-3843-11e8-874b-88d49fa83774.png)

**Comment by SqrTT** [comment]
Is `hostname` and `codeversion` correct?

**Comment by shinethazhava** [comment]
Yes previously the codes are there am using with Eclipse and now try to work with this one

**Comment by shinethazhava** [comment]
Is it possible to add me on Skype? You will get exactly what happening here.

**Comment by SqrTT** [comment]
Could you please share log from output starting from the beginning?

PS. Sorry, can't skype.

**Comment by shinethazhava** [comment]
Restarting
Using config file 'e:\macpac\macpac.git\cartridges\dw.json'
Connection validated successfully
Current active version is: shine
Start uploading cartridges
Cleanup code version...
[app_macpac_core] Deleting local zip
[app_macpac_core] Zipping
[app_macpac_core] Deleting remote zip
[app_macpac_core] Sending zip to remote
[app_macpac_core] Deleting local zip...
[app_macpac_core] Unzipping remote zip
[app_macpac_core] Deleting remote zip...
Cartridges uploaded successfully
Watching files

**Comment by SqrTT** [comment]
Are you trying to upload to an instance that requires a certificate? Can you upload code manually via WebDAV?

**Comment by SqrTT** [comment]
Currently, the extension doesn't support 2-factor auth (usually dev sandboxed doesn't require certificate) but can be added in future

**Comment by EvilJordan** [comment]
Checking in on this... did we figure it out?

**Comment by SqrTT** [comment]
Note: time is changing and currently we have two things named **2-factor auth**

* auth with additional one time  (via mobile App)
* auth which require certificates (like for Staging instance and Production)

Prophet doesn't support only second one (certificates).

**Comment by Samuel0827** [comment]
> Note: time is changing and currently we have two things named **2-factor auth**
> 
> * auth with additional one time  (via mobile App)
> * auth which require certificates (like for Staging instance and Production)
> 
> Prophet doesn't support only second one (certificates).

Do u know how to setup the certificates(2-factor auth) in Prophet?

**Comment by SqrTT** [comment]
@Samuel0827 
> Do u know how to setup the certificates(2-factor auth) in Prophet?
There are no way to setup it. Prophet doesn't support certificates for upload.
